packages/protvista-msa/src/protvista-msa.js

```diff
@@ -148,6 +150,15 @@ class ProtvistaMSA extends ProtvistaZoomable {
     );
   }
 
+  set features(_features) {
+    this._features = _features;
+    this.refresh();
+  }
+
+  handleFeatureClick(id) {
+    this.onFeatureClick(id);
```

Comment:
Maybe put a validation that `onFeatureClick`is indeed a function.

Reply:
Good call